stacks/minstack.js

```diff
@@ -0,0 +1,39 @@
+const Stack = require('./Stack');
+
+function MinStack(stackMaxSize){
+    this.stack = new Stack(stackMaxSize);
+    this.minStack = new Stack(stackMaxSize);
+}
+
+MinStack.prototype.push = function(num) {
+    var currentMinimum = this.minStack.peek();
+    if(this.minStack.count() === 0 || (!currentMinimum || currentMinimum > num) ) //First element
```

Comment:
welcome to javascript, let's learn about truthy and falsy values https://www.sitepoint.com/javascript-truthy-falsy/
!currentMinimum would be true if currentMinimum is 0

Reply:
i had assumed it to be for 1 or more number. missed a condition with 0 numbers. corrected now.